docs/meshstack.meshmarketplace.metrics-metering.md

```diff
@@ -426,9 +427,10 @@ The `metrics/samplingCounters` endpoint should return a response as shown below.
 }
 ```
 
-When calculating the cost for a reporting period, we will subtract the counter value at the beginning of the reporting period from the value at the end of the reporting period and multiply by the cost per unit. **We recommend that you sample the counter daily or hourly so that we can build incremental usage reports based on the data that is sent.** The advantage of using a `SamplingCounter` over a `PeriodicCounter` is that all calculations happen in meshStack and you only need to keep track of the metric values and the timestamps at which the values were observed.
+When calculating the cost for a reporting period, we will subtract the counter value at the beginning of the reporting period from the value at the end of the reporting period and multiply by the cost per unit. **We recommend that you sample the counter daily but at most hourly so that we can build incremental usage reports based on the data that is sent.** The advantage of using a `SamplingCounter` over a `PeriodicCounter` is that all calculations happen in meshStack and you only need to keep track of the metric values and the timestamps at which the values were observed.
 
-Any data points that have the exact same observedAt timestamp will be overwritten with the new values and any data points with an observedAt timestamp which meshStack has not yet retrieved will be inserted in meshStack. With this overwriting functionality, you have the option of correcting the past month's data points until the usage reports for the past month are finalized.
+> For SamplingCounters meshStack does not support data points to be overwritten in case they have the exact same observedAt timestamp.
+> meshStack only accepts data points to be added in a timely ordered manner, so correcting or inserting intermediate counters is **not** supported.
```

Comment:
f: just mention that you can correct a sampling counter by sending the correct value on a later request, because the final charged value will anyway be the difference between the final value for the month and the first value for the month.

Reply:
sampling counters can be corrected now